src/main/java/edu/cornell/kfs/rass/RassConstants.java

```diff
@@ -41,5 +41,7 @@ public static boolean isSuccessfulResult(RassObjectUpdateResultCode resultCode)
     public static final String PROPOSAL_ORG_CHART = "IT";
     
     public static final String RASS_MODULE = "KFS-RASS";
+    
+    public static final String RASS_FAILED_EVENT_QUEUE_READONLY = "R";
```

Comment:
This should be with the Concur constants (and named as such), not with the RASS constants.

Reply:
oh, yes I will change